newsletter/11.04.2020_newsletter.md

```diff
@@ -0,0 +1,45 @@
+---
+title: November 04, 2020 Newsletter
+permalink: /newsletter11042020/
+layout: page
+exclude: true
+---
+*This is a periodic newsletter of the interesting things we’ve seen and what we are thinking about in open source policy analysis.*
+
+**A new Tax-Cruncher Joe Biden application.** A new Tax-Cruncher
+application helps you understand how Biden’s proposed tax policies would
+influence your own financial circumstances. Kyle Pomerleau (AEI) and
+Peter Metz (AEI) created it.
+[<u>Link</u>](https://compute.studio/AEIEconomics/Tax-Cruncher-Biden/)
```

Comment:
Out of curiosity, why do we need these <u> tags?

Reply:
The `<u>` tags come up automatically after using `pandoc`.

Reply:
Whats pandoc being used for?

Reply:
So I send the newsletter as a `.docx` file to editing, and when I get it back I just use `pandoc` to convert it to markdown.

Reply:
Cool, as long as it works!